enterprise_access/apps/content_assignments/constants.py

```diff
@@ -107,3 +110,5 @@ class AssignmentAutomaticExpiredReason:
 
 
 NUM_DAYS_BEFORE_AUTO_CANCELLATION = 90
+
+RETIRED_EMAIL_ADDRESS = 'retired@edx.org'
```

Comment:
Hm, within the retirement pipeline we've gone to moderate lengths to craft a specific format for the email address.  In this case, we cannot use the exact format because we don't always even have a real LMS user to reference, but at least can we inherit the `@retired.invalid` domain? https://github.com/edx/edx-platform/blob/c233327ea07cd3a3869df532e37e7aadca13673d/lms/envs/common.py#L4853